lib/src/bluez_client.dart

```diff
@@ -657,6 +664,247 @@ class BlueZAdvertisement extends DBusObject {
   }
 }
 
+/// Management for Battery provisioning
+class BlueZBatteryProviderManager {
+  final String _batteryProviderManagerInterfaceName =
+      'org.bluez.BatteryProviderManager1';
+
+  final BlueZClient _client;
+  final _BlueZObject _object;
+  int _nextBatteryProviderId;
+  int _nextBatteryId;
+
+  BlueZBatteryProviderManager(this._client, this._object)
+      : _nextBatteryProviderId = 0,
+        _nextBatteryId = 0;
+
+  /// Registers a new battery with an optional label.
+  Future<BlueZBattery> registerBattery(String? name) async {
```

Comment:
The documentation says it's a "label", the argument is "name" and the batter object has it as "source" - I would expect these to all be the same. The documentation would then use `[name]` or similar to link to that arg.

Reply:
"source" seems like the appropriate term after reading https://github.com/hadess/bluez/blob/master/doc/battery-api.txt

Reply:
Done, source is used now.

Reply:
Added `[x]` for linking docs